API.yaml

```diff
@@ -1131,6 +1131,68 @@ paths:
             application/json:
               schema:
                 type: string
+  /experiments/{expUUID}/outputs/table/{outputID}/tooltip:
+    post:
+      tags:
+      - Virtual Tables
+      summary: API to get virtual table tooltip
+      operationId: getTooltip
+      parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : string
+          format: uuid
+      - name: outputID
+        in: path
+        description: The name of the table provider to query
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: Query parameters to fetch the timegraph tree.
+          Usually contains these parameters, but not limited to
+          - requested_items, index of the first line to query
+          - requested_times, number of lines to return
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        200:
+          description: Returns a list of tooltip entries. The returned model must be consistent, parentIds must refer to a parent which exists in the model.
```

Comment:
Do you have the concept of parent in the table?

Reply:
Also, could you define more precisely "tooltip entries"? The commit message says it can be used to retrieve raw data, but as an implementer of the protocol who reads this description and not the commit messages, I would never think of returning raw data here. Rather human-readable stuff